src/views/shipments.js

```diff
@@ -218,15 +218,25 @@ export class shipments {
     let ordered = this.ordered[this.shipment.account.to._id][genericName(transaction.drug)]
     let qty = +transaction.qty[this.role.account]
     let exp = transaction.exp[this.role.account]
+    let destMess = ordered.destroyedMessage || ""
 
     if ( ! ordered || ! qty) return
 
+    let qtyToCheck = 10   
+    if(transaction.drug.brand) qtyToCheck = 1
+
```

Comment:
minor stylistic edit: for if/else that both assign to the same variable it is better to use ternary

let qtyToCheck = transaction.drug.brand ? 1 : 10